src/Domain/FaceNormal.hpp

```diff
@@ -76,6 +80,53 @@ struct UnnormalizedFaceNormal : db::ComputeTag {
   using volume_tags = tmpl::list<ElementMap<VolumeDim, Frame>>;
 };
 
+template <size_t Dim, typename Frame>
+struct UnitFaceNormal : db::SimpleTag {
+  using type = tnsr::i<DataVector, Dim, Frame>;
+  static std::string name() noexcept { return "UnitFaceNormal"; }
+};
+
+template <size_t Dim, typename Frame>
+struct UnitFaceNormalCompute : UnitFaceNormal<Dim, Frame>, db::ComputeTag {
```

Comment:
The tag `Tags::Normalized<Tags::UnnormalizedFaceNormal<...>>` is added to the Databox on each interface and external face, and  represents the unit normal one-form. Does this tag do anything more or different to that? If it doesn't I'd say to remove it.

Reply:
@geoffrey4444 added this new compute tag, that derives from a simple tag, to have a simple tag for the unit face normal that can be used for packaging in the flux. However, now this is duplicate code, since the `Normalized` tag has the same implementation. I think we have the following options here:
- Replace the implementation in this new compute tag by an aliases to `Tags::Normalized<Tags::UnnormalizedFaceNormal>::function` and so on. Then we still have two compute tags that do the same, but at least no code duplication. Tests are needed to test the new simple tag, compute tag, and equivalence to the old compute tag. Occurrences of `Tags::Normalized<Tags::UnnormalizedFaceNormal>` should probably be updated, but I don't like that there is an ambiguity in which compute tag to use, and the old one doesn't allow retrieval by the base tag.
- Just drop the additions you made to `FaceNormal.hpp` and add a private simple tag `UnitFaceNormal` to the flux class. That's what I initially suggested, is what is done in the other fluxes, and is really all you need here to package data for the flux. Changes to `FaceNormal.hpp` can then come in a separate PR, if we decide we want them.

Reply:
Remove the changes to FaceNormal.hpp and add a base tag for `Tags::Normalized`.

Reply:
OK, I will stop using "UnitFaceNormal" etc. in my next update